packages/xstate-solid/test/useMachine.test.tsx

```diff
@@ -1672,4 +1676,77 @@ describe('useMachine (strict mode)', () => {
       jest.useRealTimers();
     }
   });
+
+  it('TODO: write a good test title here', async () => {
```

Comment:
Still being lazy, but in a more useful way 

```suggestion
  it('https://github.com/statelyai/xstate/issues/3925', async () => {
```